tests/README.md

```diff
@@ -1,64 +1,80 @@
 
-SWMM-NRTestSuite
-================
+# SWMM Regression Testing
 
-Installation
-------------
+## Prerequisits
 
-Satisfying dependencies?
-nrtest swmm was develop on Windows using Python 2.7 64 bit.  
+Running SWMM's regression test suite `swmm-nrtestsuite` requires installation of the following software. 
+* git 
+* C compiler - MSVC, gcc, xcode
+* cmake
+* python 2.7 including setup tools 
+* swig
 
-Third party package dependencies:
-header_detail_footer
-nrtest 
-numpy 
 
-Project packages: 
-nrtest_swmm
-swmm-output
+## Step by Step Guide for Linux and MacOS
 
+The following are step by step instructions to compare the current SWMM OWA build against the SWMM 5.1.12 MSVC 32 bit benchmark. 
 
-pip can be used to install local packages. 
+1. Clone the swmm github repository. 
+```
+$ git clone --depth=50 --branch=feature-nrtest https://github.com/OpenWaterAnalytics/Stormwater-Management-Model.git
+```
```

Comment:
Why `feature-nrtest` branch, we should probably just advide to do something like

```
$ git clone https://github.com/OpenWaterAnalytics/Stormwater-Management-Model.git
```

Reply:
`feature-nrtest` is currently the only branch with this code it hasn't been merged yet.

Reply:
Well in that case you should merge it, and we should have standard instructions